feature/staticroute/otg_tests/static_route_test.go

```diff
@@ -116,17 +111,16 @@ func TestGNMI(t *testing.T) {
 	ate.OTG().PushConfig(t,top)
 	ate.OTG().StartProtocols(t)	
 
-
 	flow := top.Flows().Add().SetName("Flow")
 	flow.TxRx().Device().SetTxNames([]string{"src.IPv4"}).SetRxNames([]string{"dst1.IPv4"})
 	flow.Metrics().Msg().Enable = ygot.Bool(true)
 	endpoint := flow.Packet().Add().Ipv4()
-	endpoint.Src().SetValue("192.0.2.13")
+	endpoint.Src().SetValue("192.0.2.23")
```

Comment:
Same point as above here, if you use the map, then this can become more intuitive to the user -- e.g., `atePorts["src"].IPv4`.